CONTRIBUTING.md

```diff
@@ -7,6 +7,7 @@ yourself.
 * [Code of Conduct](#coc)
 * [Filing an Issue](#issue)
 * [Building](#building)
+* [Automatic Code Formatting](#formatting)
```

Comment:
Maybe put this after testing? I feel like learning about testing is more important for newcomers.